src/components/ResultViewer.js

```diff
@@ -23,7 +24,12 @@ export class ResultViewer extends React.Component {
   static propTypes = {
     value: PropTypes.string,
     editorTheme: PropTypes.string,
+    ResultsTooltip: PropTypes.any,
   };
+  constructor() {
+    super();
+    this._tooltip = document.createElement('div');
```

Comment:
For this case, since you'd just be passing a reference of the created DOM element, I think it'll actually make the API/implementation cleaner because then you wouldn't have to "render" the DOM in here (the implementation). But yeah, it also depends on the use cases I guess